spec/javascripts/packages/document-capture/components/document-capture-spec.jsx

```diff
@@ -57,12 +57,18 @@ describe('document-capture/components/document-capture', () => {
     );
 
     initialize();
-    window.AcuantCameraUI.start.callsArgWithAsync(0, {
-      glare: 70,
-      sharpness: 70,
-      image: {
-        data: 'data:image/png;base64,',
-      },
+    window.AcuantCameraUI.start.callsFake((callbacks) => {
+      const response = {
+        glare: 70,
+        sharpness: 70,
+        image: {
+          data: 'data:image/png;base64,',
+        },
+      };
+
+      Promise.resolve()
+        .then(() => Promise.resolve(callbacks.onCaptured(response.image)))
+        .then(() => Promise.resolve(callbacks.onCropped(response)));
```

Comment:
Kinda a shame. I had grown quite fond of the expressiveness of `callsArgWithAsync` in #4122:

https://github.com/18F/identity-idp/pull/4122/files#diff-4a5c1f2d02534cc024904936a9281620R334-R352

...but we'll probably have to stick to `callsFake`, at least for the success cases.

I still think it'd be good to move this into an option of the Acuant test helper's `initialize` function.